react-app/package.json

```diff
@@ -8,73 +8,74 @@
+    "@romcal/calendar.brazil": "^3.0.0-dev.60",
+    "@romcal/calendar.canada": "^3.0.0-dev.60",
+    "@romcal/calendar.chile": "^3.0.0-dev.60",
+    "@romcal/calendar.china": "^3.0.0-dev.60",
+    "@romcal/calendar.costa-rica": "^3.0.0-dev.60",
+    "@romcal/calendar.croatia": "^3.0.0-dev.60",
+    "@romcal/calendar.czech-republic": "^3.0.0-dev.60",
+    "@romcal/calendar.denmark": "^3.0.0-dev.60",
+    "@romcal/calendar.england": "^3.0.0-dev.60",
+    "@romcal/calendar.europe": "^3.0.0-dev.60",
+    "@romcal/calendar.finland": "^3.0.0-dev.60",
+    "@romcal/calendar.france": "^3.0.0-dev.60",
+    "@romcal/calendar.france.paris": "^3.0.0-dev.60",
+    "@romcal/calendar.france.saint-denis": "^3.0.0-dev.60",
+    "@romcal/calendar.france.strasbourg": "^3.0.0-dev.60",
+    "@romcal/calendar.general-roman": "^3.0.0-dev.60",
+    "@romcal/calendar.germany": "^3.0.0-dev.60",
+    "@romcal/calendar.greece": "^3.0.0-dev.60",
+    "@romcal/calendar.guatemala": "^3.0.0-dev.60",
+    "@romcal/calendar.hungary": "^3.0.0-dev.60",
+    "@romcal/calendar.india": "^3.0.0-dev.60",
+    "@romcal/calendar.ireland": "^3.0.0-dev.60",
+    "@romcal/calendar.italy": "^3.0.0-dev.60",
+    "@romcal/calendar.japan": "^3.0.0-dev.60",
+    "@romcal/calendar.lebanon": "^3.0.0-dev.60",
+    "@romcal/calendar.lithuania": "^3.0.0-dev.60",
+    "@romcal/calendar.malta": "^3.0.0-dev.60",
+    "@romcal/calendar.mexico": "^3.0.0-dev.60",
+    "@romcal/calendar.netherlands": "^3.0.0-dev.60",
+    "@romcal/calendar.new-zealand": "^3.0.0-dev.60",
+    "@romcal/calendar.norway": "^3.0.0-dev.60",
+    "@romcal/calendar.panama": "^3.0.0-dev.60",
+    "@romcal/calendar.paraguay": "^3.0.0-dev.60",
+    "@romcal/calendar.peru": "^3.0.0-dev.60",
+    "@romcal/calendar.philippines": "^3.0.0-dev.60",
+    "@romcal/calendar.poland": "^3.0.0-dev.60",
+    "@romcal/calendar.portugal": "^3.0.0-dev.60",
+    "@romcal/calendar.puerto-rico": "^3.0.0-dev.60",
+    "@romcal/calendar.romania": "^3.0.0-dev.60",
+    "@romcal/calendar.russia": "^3.0.0-dev.60",
+    "@romcal/calendar.scotland": "^3.0.0-dev.60",
+    "@romcal/calendar.slovakia": "^3.0.0-dev.60",
+    "@romcal/calendar.slovenia": "^3.0.0-dev.60",
+    "@romcal/calendar.spain": "^3.0.0-dev.60",
+    "@romcal/calendar.sri-lanka": "^3.0.0-dev.60",
+    "@romcal/calendar.sweden": "^3.0.0-dev.60",
+    "@romcal/calendar.switzerland": "^3.0.0-dev.60",
+    "@romcal/calendar.ukraine": "^3.0.0-dev.60",
+    "@romcal/calendar.united-states": "^3.0.0-dev.60",
+    "@romcal/calendar.uruguay": "^3.0.0-dev.60",
+    "@romcal/calendar.venezuela": "^3.0.0-dev.60",
+    "@romcal/calendar.vietnam": "^3.0.0-dev.60",
+    "@romcal/calendar.wales": "^3.0.0-dev.60",
+    "@textea/json-viewer": "^3.4.0",
     "date-fns": "^2.29.3",
     "mobx": "^6.7.0",
     "mobx-react": "^7.6.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-scripts": "5.0.1",
-    "romcal": "^3.0.0-dev.57",
+    "romcal": "^3.0.0-dev.60",
```

Comment:
I have found the issue why `pnpm build` fails: the caret.

While using `dev` romcal versions, we should pin the exact romcal version, as we might (and will) introduce some breaking changes in the next `dev` version.

Reply:
interesting. i just did a replace... this is also an issue with pnpm, as base npm doesn't have this resolution issue

Reply:
Yeah, it seems to be a PNPM issue, probably related to pnpm/pnpm#6804. I presume that `^3.0.0-dev.*` resolves to any version newer than the specified version, including the stable version (which is released after the `dev` versions). For me it makes sense, although it seems to be a deviation from what NPM does.

I have no idea what resolution is used in NPM in this case.

Anyway, I still think that using exact `dev` version is a good idea. After we use a stable version, we can reintroduce the caret.

Reply:
i can definitely change it, but that's kind of unrelated to my changes... this seems like something we'd need to talk about w/ generally any other pnpm issues, cause pnpm is not the context of the example app... 

regardless, i'll get a change in in 10-12h once i'm not working.

Reply:
wasn't as free as I thought i was. it's updated now.